models/featureFlags.js

```diff
@@ -1,13 +1,14 @@
 const firestore = require('../utils/firestore')
 const featureFlagModel = firestore.collection('featureFlags')
 const userModel = require('./users')
+const { defaultConfig } = require('../constants/featureFlag')
 
 /**
  * Fetch all tasks
- *
+ * @param getOwnerInfo: Fetch the owner info for all the feature flags if true.
  * @return {Promise<featureFlags|Array>}
  */
-const fetchFeatureFlag = async () => {
+const fetchFeatureFlag = async (getOwnerInfo = true) => {
```

Comment:
Why do we want to have this option? What are the benefits of getting the `ownerInfo` of who set the flag?

Reply:
The same function can be called for doing the CRUD of feature flags.

On the internal dashboard for feature flags we can show who created the flag along with it's current status.

Ideally, I also want to track who updated as well. But beyond the scope of this PR. It can be added later.